clang/lib/Headers/emmintrin.h

```diff
@@ -4743,7 +4771,9 @@ static __inline__ __m128d __DEFAULT_FN_ATTRS _mm_castsi128_pd(__m128i __a) {
 ///    128-bit vectors of [2 x double], using the operation specified by the
 ///    immediate integer operand.
 ///
-///    Each comparison yields 0x0 for false, 0xFFFFFFFFFFFFFFFF for true.
+///    Each comparison returns 0x0 for false, 0xFFFFFFFFFFFFFFFF for true.
+///    If either value in a comparison is NaN, comparisons marked as ordered
```

Comment:
Say instead: "comparisons that are ordered return false, and comparisons that are unordered return true"? If no, I don't care for marked/unmarked. There are other instances of marked/unmarked that I have not flagged.